packages/stark-ui/src/common/classes/abstract-color.ts

```diff
@@ -0,0 +1,23 @@
+import { OnInit, Input, Renderer2, ElementRef } from "@angular/core";
+
+/**
+ * Abstract class to add the right stark class color to the elementRef
+ */
+export abstract class StarkAbstractColor implements OnInit {
```

Comment:
Could we rename this class to `AbstractStarkUiComponent`? Discussing a bit more with @SuperITMan it is a good idea that all components in Stark UI extend from it since all those components might need to be themable.